examples/rbd/snapshotclass.yaml

```diff
@@ -16,3 +16,4 @@ parameters:
 
   csi.storage.k8s.io/snapshotter-secret-name: csi-rbd-secret
   csi.storage.k8s.io/snapshotter-secret-namespace: default
+deletionPolicy: Delete
```

Comment:
Is this not the default? Maybe comment it out in that case (or mention what the default is, and why it is changed)

Reply:
this is not be default user has to provide `deletionPolicy` when creating snapshotclass

Reply:
hmm, ok